NEWS.md

```diff
@@ -0,0 +1,2 @@
+## 2022-04-29 v0.0.1
```

Comment:
I can propose the following format:
## 2022-04-29 v0.0.1 Unreleased
### Stories
* [MDEXP-458](https://issues.folio.org/browse/MDEXP-458) - Create application.yaml and setup hibernate